txsni/dehydrated.py

```diff
@@ -0,0 +1,45 @@
+"""
```

Comment:
Can you prefix this with an underscore so that it doesn't get consumed as public API?